packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -1073,3 +1089,97 @@ function rollingUpdateHelmVariables() {
     )}`,
   ]
 }
+
+const celoBlockchainDir: string = path.join(os.tmpdir(), 'celo-blockchain-celotool')
+
+export async function getGenesis(celoEnv: string, reset: boolean, useMyCelo: boolean) {
```

Comment:
nit: the seems like it's `createAndUploadGenesisToGCP` (name's a little long), but it's odd that `getGenesis` doesn't return anything but instead place the genesis in google cloud.